docs/architecture/bridges/third-party-cross-chain.mdx

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Docs under architecture should be about Linea specficially

If you want to include something about a link to a token bridge for users to bridge, it should go here: https://docs.linea.build/use-mainnet/bridges-of-linea#official-third-party-permissionless-bridges-and-beyond - I can set up an example structure

Otherwise, if you're interested in adding it as tooling for devs, it would go under here: https://docs.linea.build/build-on-linea/tooling

Feel free to create a new section if you don't feel like the existing ones fit

Or add to both